packages/crayons-core/src/components/data-table/data-table.tsx

```diff
@@ -365,7 +365,7 @@ export class DataTable {
    * @returns isLoading current state
    */
   @Method()
-  async loadTable(state: boolean) {
+  async loadTable(state = true) {
```

Comment:
Can we also have a `boolean` prop for loading, instead of just having the `method`?

Reply:
Done. Exposed isLoading state as prop as suggested.